### ISSUE TITLE
PHP opcache shm caching not working

### ISSUE THREAD

**Issue by Neo-Oli** [termux/termux-packages#2234]
Even with #2233 Opcache isn't fully working. 

```
$ php -n -dzend_extension=opcache.so -dopcache.enable_cli=1 -dopcache.lockfile_path=$PREFIX/tmp -dopcache.file_cache=$PREFIX/tmp -dopcache.file_cache_only=0

Fri Mar  9 21:48:22 2018 (547731073688): Fatal Error Unable to allocate shared memory segment of 134217728 bytes: unknown: No such file or directory (2)
```

If I disable shared memory caching with `-dopcache.file_cache_only=1` it works, but is obviously not as good.

I know shared memory is a bit weird on android. I tried linking against libandroid-shmem, but that didn't do anything.

**Comment by fornwall** [comment]
Thanks! Seems to be an issue when cross compiling, the config script assumed that no shared memory implementation was available. This should now be fixed in 7.2.3-2, does that work better?

The version also has `opcache.lockfile_path` patched to be `$PREFIX/tmp` by default.

**Comment by Neo-Oli** [comment]
Thanks. Seems to work much better.
